JASP-Engine/JASP/R/descriptives.R

```diff
@@ -1055,6 +1078,56 @@ Descriptives <- function(jaspResults, dataset, options) {
   return(p)
 }
 
+
+.descriptivesDotPlots <- function(dataset, options, variable){
+  
+  
+  if (options$splitby != "" ) {
+    # return a collection
+    split <- names(dataset)
+    
+    plotResult <- createJaspContainer(title=variable)
+    plotResult$dependOn(options="splitby", optionContainsValue=list(variables=variable))
+    
+    for (l in split) {
+      plotResult[[l]] <- .descriptivesDotPlots_SubFunc(dataset=dataset[[l]], variable=variable, title = l)
+    }
```

Comment:
```suggestion
    for (l in split)
      plotResult[[l]] <- .descriptivesDotPlots_SubFunc(dataset = dataset[[l]], variable = variable, title = l)
```
Two nitpicks:
1. Braces `{` arent necessary for a single line for loop
2. Please be consistent with spaces around arguments (e.g., `dataset = dataset[[l]]`). I know that doesn't happen everywhere in JASP, but that's a bad thing.